### ISSUE TITLE
Proper packaged release?

### ISSUE THREAD

**Issue by seb45tian** [blobtoolkit/blobtoolkit#44]
Hi,

would it be possible to create a proper packaged release, i.e. add a pyproject.toml and setup.cfg? (cf. https://packaging.python.org/tutorials/packaging-projects/ ) It does not necessarily have to be uploaded to pypi.org (although this would also be beneficial) but having a reliable way of installing blobtools via `pip install` would make life much easier and be less of hack than just cloning the repository. I know there is a Dockerfile but I want to make blobtools2 available on an HPC cluster and using containers is not really an option. 

If wanted, I can try to help with this (i.e. fork, make PRs etc.). 

Kind regards

**Comment by rjchallis** [comment]
I agree that the containers aren't ideal so it will be good to get this packaged to make installing a little easier. Thanks for offering to help - I had started on this a while back but stopped when trying to decide how integrated to make the packaging of the various components (including the pipeline and viewer) but packaging them separately should be more straightforward. I'll look back at where I had got to to save you duplicating work I've already done.

**Comment by rjchallis** [comment]
I've made a pip package of this now that includes copy of the viewer so the `blobtools view` commands can work out of the box if you `pip install blobtoolkit`.  Hope this runs happily on your cluster!
